runtime/src/append_vec.rs

```diff
@@ -136,6 +136,33 @@ impl<'a> StoredAccountMeta<'a> {
     }
 }
 
+pub struct AccountsInAppendVec<'a> {
```

Comment:
How about calling this something like `AppendVecAccountsIter`?

Reply:
done